botocore/docs/bcdoc/restdoc.py

```diff
@@ -103,7 +103,7 @@ def remove_last_doc_string(self):
 
 
 class DocumentStructure(ReSTDocument):
-    def __init__(self, name, section_names=None, target='man'):
+    def __init__(self, name, section_names=None, target='man', meta=None):
```

Comment:
Given botocore/boto3 already has this concept of `meta` which is an actual object, it would be less confusing is this was maybe called `metadata` or `context` or something besides `meta`.